scripts/alert_emails/utils.ts

```diff
@@ -85,7 +85,7 @@ function generateAlertEmailContent(
   const data: AlertTemplateData = {
     change: changeText(oldLevel, newLevel),
     location_name: locationName,
-    img_alt: `Image depicting that ${locationName} went from "${oldLevelText}" to "${newLevelText}"`,
+    img_alt: `Image depicting that ${locationName} went from "${oldLevelText}" to "${newLevelText}" (where ${oldLevelText} and ${newLevelText} can be: Low, Medium, High COVID Community Level)`,
```

Comment:
This should be just

> Image depicting that {LOCATION_NAME} went from {PREVIOUS_LEVEL} to {NEW_LEVEL}

and we need to make sure that `oldLevelText` and `newLevelText` are "Low  COVID Community Level", "Medium  COVID Community Level", "High COVID Community Level"